bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```
@@ -70,7 +70,7 @@ spec:
       serviceAccountName: {{ default (include "common.names.fullname" . ) .Values.serviceAccount.name}}
       {{- end }}
       initContainers:
-      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
+      {{- if and .Values.persistence.enabled (or (or (not (empty .Values.postgresql.extendedConf)) (not (empty .Values.postgresql.extendedConfCM)) ) .Values.volumePermissions.enabled) }}
```

We can make this simpler. Please Try with:
```suggestion
      {{- if and .Values.persistence.enabled (or (or .Values.postgresql.extendedConf .Values.postgresql.extendedConfCM) .Values.volumePermissions.enabled) }}
```